R/circlize.R

```diff
@@ -158,3 +158,6 @@ circlize_dendrogram <- function(dend, facing = c("outside", "inside"), labels =
 #  circlize_dendrogram(dend)
 #
 #
+
+
+requireNamespace <- NULL
```

Comment:
You only need to put this in one place.

Reply:
Tried to resolve it here:
https://github.com/talgalili/dendextend/commit/e094f906ad715278991bb02b59a9edae86bf24ca